providers/implementations/kdfs/hkdf.c.in

```diff
@@ -160,6 +160,9 @@ static void *kdf_hkdf_dup(void *vctx)
     const KDF_HKDF *src = (const KDF_HKDF *)vctx;
     KDF_HKDF *dest;
 
+    if (!ossl_prov_is_running())
+        return NULL;
+
     dest = kdf_hkdf_new(src->provctx);
```

Comment:
To check if this is a duplicate check, given new call is the first call in dup

Reply:
Yeah, it probably is a duplicate check.